README.md

```diff
@@ -64,59 +63,54 @@ When adding a new top-level entry-point for Toolbox (such as a new example appli
-      `build.gradle`, and the `gradlew` script.
-* From the top-level directory, run `./gradlew` to ensure that Gradle can properly configure the
-  unified build.
-* If using IntelliJ, create a new project by running through the "New project from existing
-  sources..." workflow and pointing the IDE at the top-level `settings.gradle` file.
-    * IntelliJ should detect that this is a Gradle/Grails project, download and index the
-      server-side dependencies, and set up an appropriate "Run Configuration" to start the Toolbox
-      server.
-
-Having all three repos checked out in a single IntelliJ project can be useful to have the code
-on-hand, but to actually run Toolbox using the local Hoist libraries some additional steps are
-required.
-
-* To run the server using the local `hoist-core`, edit the `toolbox/gradle.properties` file and set
-  `runHoistInline=true`. Note this is _not_ required if you're not changing any hoist-core code -
-  you can still have the project structure setup as described, and only flip this switch if/when
-  testing a local change to the plugin.
-* To run the client using the local `hoist-react`, start your local webpack-dev-server from the
-  `toolbox/client-app` directory by running `yarn startWithHoist`.
+## Toolbox + Hoist development
+
+Toolbox is often developed alongside the Hoist Core and React libraries, so that changes to the
+libraries themselves can be developed and tested locally using Toolbox as a reference app. This is
+the recommended configuration for XH developers to use when setting up Toolbox.
+
+* Create or find a suitable directory for multiple project repositories, like homedir or (if using
+  IntelliJ) `~/IdeaProjects/`.
+* Within this directory, check out the `toolbox`, `hoist-react`, and `hoist-core` repositories as
+  siblings.
+
+### Editing multiple projects together
+
+For editing Hoist Core and React alongside Toolbox, it is recommended to open the `hoist-react` and
+`hoist-core` projects as modules in your editor. Having all three repos in a single IntelliJ project
+can be useful to have the code on-hand.
+
+* Open up the `toolbox` project in IntelliJ.
+* Import `hoist-react` and `hoist-core` as modules:
+  File -> Project Structure -> Modules -> Add (+) -> Import Module
+* Using the Gradle tool window, "Link Gradle Project" (+) both `toolbox` and `hoist-core`.
+
+### Running Toolbox using local Hoist
+
+* To run the server using the local `hoist-core`, you need `hoist-core` to exist as sibling of the
+  `toolbox` package, and do one of the following:
+  * Edit the `toolbox/gradle.properties` file and set `runHoistInline=true`.
+  * Run the command `./gradlew bootRun -PrunHoistInline=true` from the `toolbox` directory.
+  * If using IntelliJ, run the `BootrunWithHoist` saved run configuration.
+
+  Note this is _only_ required if you're changing hoist-core code.
+
+* To run the client using the local `hoist-react`, you need `hoist-react` to exist as sibling of the
```

Comment:
An extension to what I have already said, but there is a lot of details here (and previously) about second tier setups (including hoist in local dev), but it maybe misses the quick, happy path setup. 

I think this is ok for now, but it may be worth a think about how we present setup to a new user, and tier the information they receive - from quick start happy path to more in depth configurations as they need it.

Reply:
Added some (at least bare bones) info about the happy path above.